images/kubekins-e2e/variants.yaml

```diff
@@ -31,6 +31,12 @@ variants:
     K8S_RELEASE: stable
     BAZEL_VERSION: 3.4.1
     OLD_BAZEL_VERSION: 2.2.0
+  '1.28':
+    CONFIG: main
```

Comment:
Should this be `1.28`?

Reply:
I think we reuse the `main` for the RC release. I replicated what was done for 1.27: https://github.com/kubernetes/test-infra/pull/29125